src/main/java/org/maxgamer/quickshop/util/language/game/MojangGameLanguageImpl.java

```diff
@@ -58,29 +61,24 @@ public MojangGameLanguageImpl(@NotNull QuickShop plugin, @NotNull String languag
         final GameLanguageLoadThread loadThread = new GameLanguageLoadThread();
         languageCode = languageCode.replace("-", "_");
         loadThread.setLanguageCode(languageCode.toLowerCase());
-        loadThread.setMainThreadWaiting(true); // Told thread we're waiting it
         loadThread.setPlugin(plugin); // Transfer instance
         loadThread.start();
         int count = 0;
-        while (count < 7) {
+        while (true) {
             if (loadThread.isAlive()) {
                 count++;
+                //noinspection BusyWait
                 Thread.sleep(1000);
```

Comment:
How about just make a condition lock here? so we can just wait a signal for 1000*20 ms, and the code is more clear.